book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ OPTIONAL ] I think the previous code reviewer didn't clarify well, so here rentals it should be attr_accessor instead of attr_reader because we want to set and get the rentals not just to get it so kindly update it 🔥💯👍🏻